taxonomy-manager-common/src/main/java/com/digirati/taxman/common/taxonomy/ConceptRelationshipType.java

```diff
@@ -49,4 +49,17 @@ public Property getSkosProperty(boolean transitive) {
 
         return transitive ? transitiveRelationshipProperty : relationshipProperty;
     }
+
+    public ConceptRelationshipType inverse() {
+        switch (this) {
+            case BROADER:
+                return NARROWER;
+            case NARROWER:
+                return BROADER;
+            case RELATED:
+                return RELATED;
+            default:
+                return null; // Or throw...
```

Comment:
Let's throw an `IllegalStateException` here telling the user that `this` can't be inversed